scripts/package.json

```diff
@@ -5,7 +5,8 @@
     "add": "node manage-versions.js -a --npm --silent",
     "clean": "node clean-data.js",
     "test": "mocha --inline-diff",
-    "watch-tests": "mocha ./**/*.test.js --inline-diff -w"
+    "watch-tests": "mocha ./**/*.test.js --inline-diff -w",
+    "generate-platform-links": "node links/generate.js"
```

Comment:
just call this `links`... make everyone happy forever.

Reply:
it's that simple? I wish I had known this earlier!